composer.json

```diff
@@ -2,7 +2,18 @@
     "name": "cakephp/cakephp",
     "description": "The CakePHP framework",
     "type": "library",
-    "keywords": ["framework"],
+    "keywords": [
+        "framework",
+        "mvc",
+        "rapid-development",
+        "conventions over configuration",
+        "dry",
+        "orm",
+        "form",
+        "validation",
+        "rest-api",
```

Comment:
I am not sure those terms should be in here -those are better used with the respective plugins that actually provide it fully.